subsys/bluetooth/services/hrs.c

```diff
@@ -91,6 +97,17 @@ static int hrs_init(void)
 	return 0;
 }
 
+int bt_hrs_cb_register(struct bt_hrs_cb *cb)
+{
+	if (cb == NULL) {
+		return -EINVAL;
+	}
```

Comment:
Why not allow unsetting it?

Reply:
I don't expect it is unset by an application, but yeah it could be. In such cases I prefer separate unset function to make the API harder to use in a wrong way e.g. one wants to setup but by accident provides NULL as the parameter.